cvat-sdk/gen/generate.sh

```diff
@@ -13,24 +13,27 @@ LIB_NAME="cvat_sdk"
 LAYER1_LIB_NAME="${LIB_NAME}/api_client"
 DST_DIR="$(cd "$(dirname -- "$0")/.." && pwd)"
 DOCS_DIR="$DST_DIR/docs"
-TEMPLATE_DIR_NAME="gen"
-TEMPLATE_DIR="$DST_DIR/$TEMPLATE_DIR_NAME"
-POST_PROCESS_SCRIPT="${TEMPLATE_DIR}/postprocess.py"
+GEN_DIR="${DST_DIR}/gen"
+POST_PROCESS_SCRIPT="${GEN_DIR}/postprocess.py"
+SCHEMA_PATH="$DST_DIR/../cvat/schema.yml"
 
 rm -f -r "$DOCS_DIR" "${DST_DIR}/${LAYER1_LIB_NAME}" \
     "${DST_DIR}/requirements/api_client.txt"
 
 # Pass template dir here
 # https://github.com/OpenAPITools/openapi-generator/issues/8420
-docker run --rm -v "$DST_DIR:/local" -u "$(id -u)":"$(id -g)" \
+docker run --rm -u "$(id -u)":"$(id -g)" \
+    -v "$SCHEMA_PATH:/mnt/schema.yml:ro" \
```

Comment:
Is there a reason why you prefer this substitution style? Could you please use `${FOO}` everywhere?

Reply:
Updated.